test/test.c

```diff
@@ -34440,6 +34513,7 @@ static testInfo allTests[] =
     {"SyncSubscriptionPending",         test_SyncSubscriptionPending},
     {"SyncSubscriptionPendingDrain",    test_SyncSubscriptionPendingDrain},
     {"AsyncErrHandler",                 test_AsyncErrHandler},
+    {"AsyncSetErrHandler",              test_AsyncSetErrHandler},
```

Comment:
This is not enough to add a test to the test suite. Follow instructions [here](https://github.com/nats-io/nats.c#testing).